test/test_browser.py

```diff
@@ -77,13 +87,13 @@ def test_get_newest_pages(self):
     def test_get_genres(self):
         "get_genres returns genere list"
         genre_list = self.browser.get_genres()
-        self.assertGreater(len(genre_list), 10)
-        self.assertEqual(genre_list[0], {
-            'url': 'genre/adventure/1',
+        self.assertGreater(len(genre_list), 0)
+        self.assertIn({
```

Comment:
right, i only wanted to check if the genre list contains anything as the content might change